### ISSUE TITLE
Pedantic Photovoltaics

### ISSUE THREAD

**Issue by thorade** [UdK-VPT/BuildingSystems#21]
```
Advanced.PedanticModelica:=true;
checkModel("BuildingSystems.Technologies.Photovoltaics");
```

**Comment by carlesRT** [comment]
Errors are

I. Output connector using the type prefix parameter (same occured in #22): solved in bd4b932

II. Incompatible connectors
```Modelica
Connecting different kinds of variables
when connecting input radiationPort.IrrDir and opticalModel.radiationPort.IrrDir.
```
There is no need to use the prefix input in the connector. Solved in d5b33a8

III. Incompatible units
```Modelica
Incompatible units in
electricalModel.Ul = electricalModel.Ul0*(1.0+electricalModel.alphaUl*( electricalModel.T-298.15));
```

change alphaUl type from `Real` to `Modelica.SIunits.LinearTemperatureCoefficient`. It was also necessary to change units of `tUl0`. see 6a025a4


Unsolved problems are non well-posed models and structurally singular models.

**Comment by nytschgeusen** [comment]
Issue solved
